indico/modules/events/registration/util.py

```diff
@@ -187,6 +187,13 @@ def get_flat_section_submission_data(regform, *, management=False, registration=
     return {'sections': section_data, 'items': item_data}
 
 
+@make_interceptable
+def can_see_section(section, user, is_management):
+    if user is None:
+        return False
+    return bool(not section.is_manager_only or is_management)
```

Comment:
Not necessary, it's already a bool

```suggestion
    return not section.is_manager_only or is_management
```